spec/integration/taxonomy_spec.rb

```diff
@@ -0,0 +1,67 @@
+require "spec_helper"
+require "flex_commerce_api"
+require "uri"
+
+RSpec.describe FlexCommerce::Taxonomy do
```

Comment:
Wondering if we need to add specs for #index, #create and #update endpoints

Reply:
There's no routes for it so I guess not.